docs/docs/general/01-Introduction/why-askui.md

```diff
@@ -3,16 +3,11 @@ sidebar_position: 1
 ---
 
 # Why askui?
-
-The askui library provides you with an API that allows you to automate the interaction with an application, multiple applications or even an entire operating system (OS). You can use this to write ui, e2e or functional tests.
+The askui library is a Typescript SDK and API which automates interactions with an application, multiple applications, or an entire operating system (OS) allowing you to quickly and easily write UI, e2e, and functional tests in the same environment or across environments concurrently.
```

Comment:
I think it may be hard to understand what `same environment or across environments` actually means. Maybe an example would do good. As this is really an edge case of the enumeration `application, multiple applications, or an entire operating system`, maybe we should combine the two. Also I don't get `concurrently` part here. I think it may be misunderstood, especially by someone with a computer science background.

Just an idea:

```suggestion
The askui library is a Typescript SDK which allows you to automate interactions with a device or even multiple devices. For example, 

- you could write an email inside your favorite note-taking app, then, 
- copy over the email to your email client running in your web browser to send it, and, then, 
- head over to your mobile phone to check that the email is inside the `Sent` folder of your email client's installation on your mobile phone - 

completely automatically and inside one short script. 

This way, you can write UI, e2e and functional tests more powerful than ever without caring about underlying technology or application contexts. 
```